ironic-python-agent/0001-Bumping-version-of-debian-to-stretch-adding-user-cor.patch

```diff
@@ -0,0 +1,124 @@
+From 2fe7260df7dc6eecbc82751704840a98c860dd0e Mon Sep 17 00:00:00 2001
+From: PiotrProkop <piotr.prokop@intel.com>
+Date: Thu, 11 Jan 2018 13:34:44 +0100
+Subject: [PATCH] Bumping version of debian to stretch, adding user core to
+ coreos IPA image. Fixing script coreos-oem-inject.py to work on machines with
+ Python3 as default interpreter.
+
+---
+ Dockerfile                             | 20 ++++++++++++++------
+ get_megacli.sh                         | 16 ++++++++++++++++
+ imagebuild/coreos/coreos-oem-inject.py |  3 ++-
+ imagebuild/coreos/oem/cloud-config.yml |  5 +++++
+ 4 files changed, 37 insertions(+), 7 deletions(-)
+ create mode 100755 get_megacli.sh
+
+diff --git a/Dockerfile b/Dockerfile
+index 8c7a56a..d8f02b4 100644
+--- a/Dockerfile
++++ b/Dockerfile
+@@ -1,4 +1,4 @@
+-FROM debian:jessie
++FROM debian:stretch
+ 
+ # The add is before the RUN to ensure we get the latest version of packages
+ # Docker will cache RUN commands, but because the SHA1 of the dir will be
+@@ -13,20 +13,25 @@ ADD . /tmp/ironic-python-agent
+ # 1.6. Using the ARG command will be a much cleaner solution.
+ COPY proxy.sh /usr/bin/proxy.sh
+ 
++ADD get_megacli.sh /tmp/get_megacli.sh
++
+ # Ensure we hit a single mirror for builds, since httpredir is flakey
+ RUN sed -i 's/httpredir/http.us/g' /etc/apt/sources.list
+ 
+ # Add 'backports' for qemu-utils
+-RUN echo 'deb http://http.us.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/backports.list
++RUN echo 'deb http://http.us.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/backports.list
++
++ENV DEBIAN_FRONTEND noninteractive
+ 
+ # Install requirements: Python for ironic-python-agent, others for putting an
+ # image on disk
+ RUN proxy.sh apt-get update && \
+     proxy.sh apt-get -y upgrade && \
+-    proxy.sh apt-get install -y --no-install-recommends gdisk python2.7 python2.7-dev \
+-        python-pip qemu-utils parted hdparm util-linux genisoimage git gcc \
+-        bash coreutils tgt dmidecode ipmitool psmisc dosfstools && \
+-    proxy.sh apt-get --only-upgrade -t jessie-backports install -y qemu-utils
++    proxy.sh apt-get install -y --no-install-recommends gdisk python2.7 python2.7-dev python2.7-setuptools lvm2 \
++        python-pip qemu-utils parted hdparm util-linux genisoimage git gcc udev mdadm wget unzip alien libguestfs-tools \
++        bash coreutils tgt dmidecode ipmitool psmisc dosfstools pciutils && \
```

Comment:
nit: Can we reformat this whole command so that the stuff we're adding is all at the end, rather than adding to each line a little bit?

Reply:
+1